glusterd2/store/store.go

```diff
@@ -18,7 +19,10 @@ import (
 )
 
 const (
-	sessionTTL = 30 // used for etcd mutexes and liveness key
+	sessionTTL    = 30 // used for etcd mutexes and liveness key
+	getTimeout    = 5 * time.Second
```

Comment:
I'm sorry, I should've noticed this before. Define this as constant and move the `time.Second` to place where it's used.

```go
context.WithTimeout(context.Background(), getTimeout * time.Second)
```